druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/InMemoryDBLookupExtractor.java

```diff
@@ -90,6 +93,7 @@ public String apply(@NotNull String val)
             }
 
             if (cacheByteValue == null || cacheByteValue.length == 0) {
+                kafkaManager.handleMissingLookup(extractionNamespace, dimension);
```

Comment:
Do you want to do a local cache which has the most recent request missing lookups so you don't do multiple writes for the same key ?